apple/Sources/Sargon/Extensions/Methods/RET/TransactionManifest+Wrap+Functions.swift

```diff
@@ -1,5 +1,14 @@
-extension TransactionManifest: CustomStringConvertible {
-	public var description: String {
-		transactionManifestToString(manifest: self)
+extension TransactionManifest {
```

Comment:
Ah, actually, can you bring back `extension TransactionManifest: CustomStringConvertible` and let it be

```swift
extension TransactionManifest: CustomStringConvertible {
     public var description: String { instructionsString }
}
```